Lib/test/test_genexps.py

```diff
@@ -103,7 +103,7 @@
     >>> dict(a = i for i in range(10))
     Traceback (most recent call last):
        ...
-    SyntaxError: invalid syntax
+    SyntaxError: invalid syntax. Maybe you meant '==' or ':=' instead of '='?
```

Comment:
I'm not sure the suggestion here is very useful. More likely some other misunderstanding or typo happened.

Reply:
In this particular example yes, but imagine something like `all(x = y for x in elements for y in other_elements)`. As it makes more sense depending on the actual function, is going to be hard to contextualize.